Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -86,7 +87,7 @@ public NetworkManager(IConfigurationManager configurationManager, ILogger<Networ
             _interfaces = new List<IPData>();
             _macAddresses = new List<PhysicalAddress>();
             _publishedServerUrls = new Dictionary<IPData, string>();
-            _eventFireLock = new object();
+            _networkEvent = new SemaphoreSlim(1, 1);
```

Comment:
When it's not async there's not really any reason to use a semaphoreslim instead of a lock